hack/generate.sh

```diff
@@ -68,19 +68,43 @@ client-gen --clientset-name versioned \
 find ${KUBEVIRT_DIR}/pkg/ -name "*generated*.go" -exec rm {} -f \;
 
 ${KUBEVIRT_DIR}/hack/build-go.sh generate ${WHAT}
+
+# Genearte validation with controller-gen and create go file for them
+(
+    cd ${KUBEVIRT_DIR}/staging/src/kubevirt.io/client-go &&
+        # supress -mod=vendor
+        GOFLAGS= controller-gen crd paths=./api/v1/
+    #include snapshot
+    GOFLAGS= controller-gen crd paths=./apis/snapshot/v1alpha1/
+
+    cd config/crd
+    # tail -n +3 kubevirt.io_kubevirts.yaml > kubevirt.io_kubevirts_new.yaml
+    for file in *; do
+        tail -n +3 $file >$file"new"
+        mv $file"new" $file
+    done
+    cd ${KUBEVIRT_DIR}/tools/crd-validation-generator/ && go_build
+
+    cd ${KUBEVIRT_DIR}
+    ${KUBEVIRT_DIR}/tools/crd-validation-generator/crd-validation-generator -crdDir=${KUBEVIRT_DIR}/staging/src/kubevirt.io/client-go/config/crd/
```

Comment:
since we're setting the crdDir, would it make sense to explicitly set the output dir here as an argument as well?